src/solidClientHelpers/profile.js

```diff
@@ -94,16 +94,15 @@ export function packageProfile(webId, dataset, pods, inbox) {
 }
 
 export async function fetchProfile(webId, fetch) {
-  const profiles = await getProfileAll(webId, { fetch });
-  const {
-    webIdProfile,
-    altProfileAll: [altProfile],
-  } = profiles;
+  const profiles = await getProfileAll(webId);
```

Comment:
Omitting the fetch here is not correct but it does happen to make Pod Browser able to add a contact. This will technically "fix" the error but there would be more work to fix it correctly. Having said that, it's my understanding that contacts are deprecated in Pod Browser so I'm not sure how much attention this is worth aside from preventing the error.